Content.Server/Botany/Systems/MutationSystem.cs

```diff
@@ -142,17 +103,8 @@ public SeedData Cross(SeedData a, SeedData b)
     // Mutate reference 'val' between 'min' and 'max' by pretending the value
     // is representable by a thermometer code with 'bits' number of bits and
     // randomly flipping some of them.
-    //
-    // 'totalbits' and 'mult' are used only to calculate the probability that
-    // one bit gets flipped.
-    private void MutateFloat(ref float val, float min, float max, int bits, int totalbits, float mult)
+    public void MutateFloat(ref float val, float min, float max, int bits)
```

Comment:
Is the only caller in `PlantChangeStat`? If so, it might make sense to move this function there and keep it private.